anndata/readwrite/utils.py

```diff
@@ -195,3 +196,19 @@ def func_wrapper(elem, key, val, *args, **kwargs):
             ) from e
 
     return func_wrapper
+
+
+def _read_legacy_raw_into(
+    f, raw, read_df, read_attr, *, attrs=("X", "var", "varm")
+):
```

Comment:
Why does this modify a dict instead of creating one?

Reply:
temporary, internal, needs to make sure the dict it receives is empty anyway.

But sure, it’s still possible to do this pythonically